azurerm/config.go

```diff
@@ -397,6 +398,13 @@ func getArmClient(c *authentication.Config) (*ArmClient, error) {
 	cgc.SkipResourceProviderRegistration = c.SkipProviderRegistration
 	client.containerGroupsClient = cgc
 
+	cmc := containerservice.NewManagedClustersClientWithBaseURI(endpoint, c.SubscriptionID)
```

Comment:
Can you modify this to the new registration scheme ([see here](https://github.com/sozercan/terraform-provider-azurerm/blob/9e5d8cb6a5b1579c9eeb134cdae1a42beeefb345/azurerm/config.go#L618))

Reply:
Thanks, added managedCluster and containerService to new registration schema